swf/src/avm2/types.rs

```diff
@@ -139,7 +139,7 @@ pub struct Exception {
     pub from_offset: u32,
     pub to_offset: u32,
     pub target_offset: u32,
-    pub variable_name: Index<String>,
+    pub variable_name: Index<Multiname>,
```

Comment:
Could you add a comment that this is an erratum in the AVM2 overview?

Reply:
Added.